orm/orm_go18.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build go1.8
```

Comment:
I don't think we need to rename the file. For travis, it's already change to go1.9

Reply:
ok, I've pushed a commit to rename it back